.travis.yml

```diff
@@ -13,3 +13,14 @@ addons:
     packages:
       - g++-4.8
```

Comment:
Do we still need this?

Reply:
@rprieto I'll remove it and see if the build passes

Reply:
@rprieto the build passes 👍